core/controllers/reader.py

```diff
@@ -213,6 +213,25 @@ def get(self, exploration_id):
 
         exploration_rights = rights_manager.get_exploration_rights(
             exploration_id, strict=False)
+
+        # Retrieve all classifiers for the exploration.
+        state_classifier_mapping = {}
+        classifier_training_jobs = (
+            classifier_services.get_classifier_training_jobs(
+                exploration_id, exploration.version, exploration.states))
+        for index, state_name in enumerate(exploration.states):
+            if classifier_training_jobs[index] == None:
+                state_classifier_mapping[state_name] = None
+            else:
+                # Not making this a multi-get since we are deprecating
```

Comment:
@seanlip We are already doing get_multi() to get all jobs. After retrieving jobs, we retrieve ClassifierData models too. We need ClassifierData because we don't know the classifier_data yet. That is what we will be passing to the front-end. We plan to keep classifier_data as a field of the job. So, when we do that, the get_multi() on the job will be sufficient.